rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -268,11 +269,6 @@ impl TunnelManager {
             })
             .filter_map(|(ip_address, dev)| {
                 info!("neighbor at interface {:?}, ip {}", dev, ip_address,);
-                if let Some(dev) = dev.clone() {
```

Comment:
@jkilpatr any reason you removed this?

Reply:
this is where we filter out entries we don't need right?

Reply:
yeah, but why was it removed? so now we are only triggereing neighbor disc on listen interfaces but we connect on all interfaces

Reply:
I'll need to look at this more carefully. Do you think it's causing the connection loss problem?